### DISCUSSION TITLE
What are best practices for generating a sitemap in Remix?

### DISCUSSION THREAD

**Discussion by 684efs3** [remix-run/remix#8232] [Q&A]
What are best practices for generating a sitemap in Remix? For example, in NextJs there is an excellent sitemap package (https://www.npmjs.com/package/next-sitemap). Is there something comparable for Remix? 

This would be great for Remix to have. Or is this something that can be done with a Vite plugin?

**Comment by kiliman** [comment]
Take a look at Epic Stack. It has sitemaps.

https://github.com/epicweb-dev/epic-stack/blob/main/app/routes/_seo%2B/sitemap%5B.%5Dxml.ts
